### PR TITLE
Improving the way linking parents works

### PR DESCRIPTION
**Pull request by corps**
Bug found while working with @dtm2451 , pushed this to production to validate.

Basically, the intention was that models that need to hang off the project should automatically get the project name filled in to save some convenience, but a typo prevented that.

Also included is the ability to override the single cell linker's model parent linking process.  Ideally, in the future, a naming service would allow airflow to pass a model name and a record name, and get back all the parent model / attributes that should be derived from it.  For now, this has to be managed project by project since there is not consistency in the lineage.

### CODE REVIEW

File: airflow/opt/providers/etna/etna/etls/metis.py [hunk 1]
```diff
@@ -243,7 +244,7 @@ def _expand_project_parents(self, update: UpdateRequest, models: Dict[str, Model
                 continue
             model = models[model_name]
             self.log.info(f"Expanding project={project} for model {model_name}...")
-            if model.template.parent == "parent":
+            if model.template.parent == "project":
```
Comment (dtm2451) [inline airflow/opt/providers/etna/etna/etls/metis.py:247]: oooh is this where my xneo1 linking went wrong?

**Review by dtm2451 — APPROVED** [review]

File: airflow/opt/providers/etna/etna/etls/metis.py (hunk 1 above)
Comment (corps) [reply airflow/opt/providers/etna/etna/etls/metis.py:247]: Yessir!

**Review by coleshaw — APPROVED** [review]

**Review by dtm2451 — CHANGES_REQUESTED** [review]
Ooh, I think I just found another call that you make to `create_and_link_parents` which should be changed to just `link_parents`.  I couldn't highlight in the 'files changed' tab cuz there's nothing nearby currently, but looks like line 94, in the section with TaskGroup(f"{model_name}_processed"):

**Comment by corps** [comment]
> Ooh, I think I just found another call that you make to `create_and_link_parents` which should be changed to just `link_parents`. I couldn't highlight in the 'files changed' tab cuz there's nothing nearby currently, but looks like line 94, in the section with TaskGroup(f"{model_name}_processed"):

You are correct, addressing now.

**Pull request merged**